doc: Add local test and doc to CONTRIBUTING guide (#783)
ansys/pyprimemesh: CONTRIBUTING.md

# Contribute

Overall guidance on contributing to a PyAnsys library appears in the
[Contributing] topic in the *PyAnsys developer's guide*. Ensure that you
are thoroughly familiar with this guide before attempting to contribute to
PyPrimeMesh.

The following contribution information is specific to PyPrimeMesh.

> [Contributing]: https://dev.docs.pyansys.com/how-to/contributing.html

Reviewer: @AlejandroFernandezLuces Why this link here? I think it should be removed.

Author: It looks like this once rendered, it's just to not have the link in the middle of the text: 
![image](https://github.com/ansys/pyprimemesh/assets/17165476/77cbcc1c-75e2-4926-993f-ebfaf9a83b9f)

## Run tests locally

> To run tests locally, run the following command:

Reviewer: ```suggestion
Run tests locally with this command:
```
I didn't like the use of "run" twice in such a short sentence. It was awkward.

> Repositories should have a contributing section, but we don't need to have all the contributing information here.  Have them reference the developer guide.

Reviewer: ```suggestion
Repositories should have a contributing section, but we don't need to have all the contributing information here.  You should reference the [Contributing](https://dev.docs.pyansys.com/how-to/contributing.html) topic in the *PyAnsys developer's guide*.
```

> We absolutely welcome any code contributions and we hope that this

Reviewer: ```suggestion
We welcome any code contributions and hope that this
```

> guide will facilitate an understanding of the PyPrimeMesh code

Reviewer: ```suggestion
guide facilitates an understanding of the PyPrimeMesh code
```

> guide will facilitate an understanding of the PyPrimeMesh code
> repository. It is important to note that while the PyPrimeMesh software
> package is maintained by ANSYS and any submissions will be reviewed
> thoroughly before merging, we still seek to foster a community that can
> support user questions and develop new features to make this software

Reviewer: ```suggestion
guide facilitates an understanding of the PyPrimeMesh code. It is important to
note that while the PyPrimeMesh package is maintained by Ansys and submissions are reviewed
thoroughly before merging, we still seek to foster a community that can
support user questions and develop new features to make this software
```

> Please reference the [PyAnsys Developer's
> Guide](https://dev.docs.pyansys.com/) for the full documentation

Reviewer: ```suggestion
See the [PyAnsys developer's
guide](https://dev.docs.pyansys.com/) for the full documentation
```

This whole paragraph doesn't make sense because the dev guide isn't about contributing to PyPrimeMesh. Plus, it is redundant with the earlier information on contributing. I know this content is from a template, but that template should be revised to follow our style guidelines.

Author: You are right, it seems that this template was out of date. I updated to the latest template that appears in this link:

https://dev.docs.pyansys.com/packaging/structure.html#the-contributing-md-file

It should be better now, if you see any mistakes here, I'll open an issue in the appropiate repo 🙂

regarding contributing to the PyPrimeMesh project.

> ## Running the tests locally

Reviewer: ```suggestion
## Run tests locally
```

> To run the tests, you can simply run the following command in your local:

Reviewer: ```suggestion
To run the tests locally, you can run the following command:
```

```bash

>     pytest

Reviewer 2: @AlejandroFernandezLuces have you tried locally running `pytest`

Reviewer 2: I tried it myself.. it is working

Author: Thanks for testing it 😄

```

Note that you need to have access to PyPrimeMesh product to run the tests.
Some of the graphical related tests need a previously generated image cache, so
you might get errors due to this.

> ## Building the documentation

Reviewer: ```suggestion
## Build documentation
```

> To build the documentation of the project, you must run the following command.

Reviewer: ```suggestion
To build PyPrimeMesh documentation, run this command.
```
